addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -288,4 +288,25 @@ patch(PosStore.prototype, {
     toggleEditMode() {
         this.isEditMode = !this.isEditMode;
     },
+    async updateModelsData(models_data) {
+        const floors = models_data["restaurant.floor"];
+        if (floors) {
+            this.floors = floors;
+            this.loadRestaurantFloor();
+            const result = await this.orm.call(
+                "pos.config",
+                "get_tables_order_count_and_printing_changes",
+                [this.config.id]
+            );
+            for (const table of result) {
+                const table_obj = this.tables_by_id[table.id];
+                if (table_obj) {
+                    table_obj.order_count = table.orders;
+                    table_obj.changes_count = table.changes;
+                    table_obj.skip_changes = table.skip_changes;
+                }
+            }
+        }
+        super.updateModelsData(models_data);
```

Comment:
```suggestion
        return super.updateModelsData(models_data);
```
To make sure it is awaited.